lldap_graphql/__init__.py

```
@@ -0,0 +1,204 @@
+
+from .const import (
+    HTTP_AUTH_ENDPOINT,
+    HTTP_GRAPHQL_ENDPOINT,
+    HTTP_REFRESH_ENDPOINT,
+    LLDAP_HTTPURL_DEFAULT,
+)
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+class lldap_graphql:
```

Pylint (3.3.7)

[convention] Class name "lldap_graphql" doesn't conform to PascalCase naming style

(C0103)